pkg/storage/common/common.go

```diff
@@ -777,6 +771,19 @@ func IsNonDistributable(mediaType string) bool {
 		mediaType == ispec.MediaTypeImageLayerNonDistributableZstd //nolint:staticcheck
 }
 
+func IsEmptyLayersError(err error) bool {
+	var validationErr schema.ValidationError
+	if errors.As(err, &validationErr) {
+		if len(validationErr.Errs) == 1 && strings.Contains(err.Error(), "layers: Array must have at least 1 items") {
```

Comment:
Define this error separately. Also can we add a test (if we don't have one already) where there are multiple validation errors, and this is just 1 of them?

Reply:
added in TestArtifact in controller_test.go

what do you mean to define it separately? it's defined in schema package

Reply:
I see a hardcoded string `"layers: Array must have at least 1 items"`.
That's what I meant.